src/test/java/org/codehaus/plexus/util/cli/CommandlineTest.java

```diff
@@ -94,11 +99,20 @@ public void testExecute()
         throws Exception
     {
         // allow it to detect the proper shell here.
+        String binary = "echo";
         Commandline cmd = new Commandline();
-        cmd.setWorkingDirectory( baseDir );
-        cmd.setExecutable( "echo" );
-        assertEquals( "echo", cmd.getShell().getOriginalExecutable() );
+        cmd.setWorkingDirectory( baseDir );            
```

Comment:
Trailing WS

Reply:
Coding using copy/paste without configuring project code formatting is definitevely bad, shame on me 🙄  🔫